ppr-ui/src/store/mutations/mutations-model.ts

```diff
@@ -369,7 +370,7 @@ export const mutateMhrBaseInformation = (state: StateIF, { key, value }) => {
 }
 
 export const mutateMhrSubmittingParty = (state: StateIF, { key, value }) => {
-  state.stateModel.mhrRegistration.submittingParty[key] = value
+  set(state.stateModel.mhrRegistration.submittingParty, key, value)
```

Comment:
What makes this mutation unique from the rest?

Reply:
Using lodash set() we could set nested properties for `submittingParty`, e.g. a`ddress.country `

Reply:
So this is required to set `address.country` ? 
Fair enough if so! 

I guess there are pros and cons to trying to minimize the actions and mutations, so it seems like this is a good solution/alternative 👍

Reply:
Yea this is needed, otherwise we won't be able to set nested props, like `country` only.